renovate/config.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+    repositories: [
```

Comment:
add hello-world operator here (and maybe sort the list)?

Reply:
Both good points. Done.